test/basic.html

```diff
@@ -206,6 +206,211 @@
         });
       });
 
+      suite('accessibility', function() {
+        var LEFT = 37;
+        var RIGHT = 39;
+        var tabs;
+
+        setup(function () {
+          tabs = fixture('basic');
+          Polymer.dom.flush();
+        });
+
+        test('paper-tabs has role tablist', function() {
+          assert.equal(tabs.getAttribute('role'), 'tablist');
+        });
+
+        test('paper-tab has role tab', function() {
+          tabs.items.forEach(function(tab) {
+            assert.equal(tab.getAttribute('role'), 'tab');
+          });
+        });
+
+        test('tabs are selected when moved to using arrow keys', function(done) {
+          ensureDocumentHasFocus();
+          Polymer.Base.async(function() {
+            tabs.select(0);
+            MockInteractions.pressAndReleaseKeyOn(tabs.selectedItem, RIGHT);
+            Polymer.Base.async(function() {
+              assert.equal(tabs.selected, 1);
+
+              MockInteractions.pressAndReleaseKeyOn(tabs.selectedItem, RIGHT);
+              Polymer.Base.async(function() {
+                assert.equal(tabs.selected, 2);
+
+                MockInteractions.pressAndReleaseKeyOn(tabs.selectedItem, LEFT);
+                Polymer.Base.async(function() {
+                  assert.equal(tabs.selected, 1);
+                  done();
+                }, 1000);
```

Comment:
Why doesn't `1` work here? I.e. why does it need to be 1000, and this test to take.. 3 seconds to run :(